packages/core/src/ceramic.ts

```diff
@@ -354,7 +354,7 @@ class Ceramic implements CeramicApi {
     const ipfsTopology = new IpfsTopology(ipfs, networkOptions.name, logger)
     const pinStoreFactory = new PinStoreFactory(ipfs, pinStoreOptions)
     const repository = new Repository(streamCacheLimit, concurrentRequestsLimit, logger)
-    const dispatcher = new Dispatcher(ipfs, networkOptions.pubsubTopic, repository, logger, pubsubLogger)
+    const dispatcher = new Dispatcher(ipfs, networkOptions.pubsubTopic, repository, logger, pubsubLogger, loggerProvider.makeServiceLogger('dispatcher'))
```

Comment:
I wonder if we should call this logger `ipfs`?  With the plan to one day pull out the functionality for loading/storing from IPFS out of the Dispatcher into its own class that's explicitly just for interacting with the IPFS dag.